deployment/ingress/eks/ingress.yaml

```diff
@@ -18,7 +18,7 @@ spec:
             pathType: Prefix
             backend:
               service:
-                name: ${k8s_namespace}-service
+                name: fuel-core-service
```

Comment:
Where does this name come from? It looks like base/service.yaml uses `fuel-core-k8-lb`